<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
color: #000000;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QFrame" name="SideMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>200</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QListWidget" name="SideList">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>13</y>
       <width>186</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="Window">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>600</width>
      <height>500</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="Plays">
     <widget class="QStackedWidget" name="playsWidget">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>25</y>
        <width>550</width>
        <height>450</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="PlaysListWidget" name="playsListWidget"/>
      <widget class="PlayCard" name="playCard"/>
     </widget>
    </widget>
    <widget class="QWidget" name="Games">
     <widget class="QStackedWidget" name="gamesWidget">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>25</y>
        <width>550</width>
        <height>450</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ItemList" name="gamesListWidget"/>
      <widget class="GameCard" name="gameCard"/>
     </widget>
    </widget>
    <widget class="QWidget" name="Players">
     <widget class="QStackedWidget" name="playersWidget">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>25</y>
        <width>550</width>
        <height>450</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="ItemList" name="playersListWidget"/>
      <widget class="PlayerCard" name="playerCard"/>
     </widget>
    </widget>
    <widget class="QWidget" name="Statistics">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Статистика</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings">
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Настройки</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ItemList</class>
   <extends>QWidget</extends>
   <header>./general/itemlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameCard</class>
   <extends>QWidget</extends>
   <header>./gamecard/gamecard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayerCard</class>
   <extends>QWidget</extends>
   <header>./playercard/playercard.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlaysListWidget</class>
   <extends>QWidget</extends>
   <header>./playstab/playslistwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayCard</class>
   <extends>QWidget</extends>
   <header>./playstab/playcard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
